components/public-api/gitpod/v1/prebuild.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package gitpod.v1;
+
+import "google/protobuf/timestamp.proto";
+import "gitpod/v1/pagination.proto";
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/v1";
+
+service PrebuildService {
+    rpc StartPrebuild(StartPrebuildRequest) returns (StartPrebuildResponse) {}
+    rpc StopPrebuild(StopPrebuildRequest) returns (StopPrebuildResponse) {}
+
+    rpc GetPrebuild(GetPrebuildRequest) returns (GetPrebuildResponse) {}
+    rpc ListPrebuilds(ListPrebuildsRequest) returns (ListPrebuildsResponse) {}
+    rpc WatchPrebuild(WatchPrebuildRequest) returns (stream WatchPrebuildResponse) {}
+}
+
+message GetPrebuildRequest {
+    string prebuild_id = 1;
+    string workspace_id = 2;
+    string configuration_id = 3;
+    string branch = 4;
+}
+message GetPrebuildResponse {
+    Prebuild prebuild = 1;
+}
+
+message ListPrebuildsRequest {
+    PaginationRequest pagination = 1;
+    string prebuild_id = 2;
+    string configuration_id = 3;
+    string branch = 4;
+}
+
+message ListPrebuildsResponse {
+    PaginationResponse pagination = 1;
+    repeated Prebuild prebuilds = 2;
+}
+
+message StartPrebuildRequest {
+    string configuration_id = 1;
+    string branch = 2;
+}
+message StartPrebuildResponse {
+    string prebuild_id = 1;
+}
+
+message StopPrebuildRequest {
+    string prebuild_id = 1;
+    // waht for?
+    string configuration_id = 2;
+}
+message StopPrebuildResponse {
+}
+
+message WatchPrebuildRequest {
+    string prebuild_id = 1;
+    string workspace_id = 2;
```

Comment:
This is the "workspace this prebuild is builtd with", right? :thinking:  ([ref](https://github.com/gitpod-io/gitpod/pull/19062/files#r1390766274))

Reply:
yeah, it is for backward compatibility to the dashboard. I can add a deprecation warning. I can rename it to `build_workspace_id`?